components/PassengerHome.js

```diff
@@ -1,10 +1,95 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { MapView, Constants, Location, Permissions } from 'expo';
+import {  StyleSheet, Text, Dimensions, View, Platform, Alert } from 'react-native';
+import { Button } from 'react-native-elements';
+import firestore from '../firestore';
+import { Marker, PROVIDER_GOOGLE } from 'react-native-maps';
+import { firebase } from '@firebase/app';
+const key = 'AIzaSyDVmcW1my0uG8kBPgSHWvRhZozepAXqL_A';
+import getDirections from 'react-native-google-maps-directions'
+import LotBanner from './LotBanner'
 import Drawer from './Drawer';
 
-export default class PassengerHome extends React.Component{
-	render(){
-		return(
-			<Drawer />
-		)
+
+export default class PassengerHome extends Component {
+
+  state = {
+    location: null,
+    errorMessage: null,
+    marker: { latitude: null, longitude: null },
+    showBanner: false
+  }
+
+  componentDidMount() {
+    this._getLocationAsync()
+  }
+
+  _getLocationAsync = async () => {
+    let { status } = await Permissions.askAsync(Permissions.LOCATION);
+    if (status !== 'granted') {
+      this.setState({
+        errorMessage: 'Permission to access location was denied',
+      });
+    }
+
+    let location = await Location.getCurrentPositionAsync({});
+    this.setState({ location });
+  };
+
+  handleSubmit = async () => {
+      this.setState({ showBanner: true })
 	}
+
+  render(){
+    const { location, marker, showBanner } = this.state;
+    return(
+      <View style={styles.container}>
+      <Drawer />
+      <MapView style={styles.map}
+        onRegionChangeComplete={this.onRegionChangeComplete}
+        showsUserLocation={true}
+        followsUserLocation={true}
+        onRegionChangeComplete={this.onRegionChangeComplete}>
+        {marker.latitude ? <Marker
+          coordinate={marker}
+        /> : null}
+      </MapView>
+
+      <Button
+            title="Where to?"
+            style={styles.button}
+            backgroundColor='white'
+            color='grey'
+            onPress={this.handleSubmit} />
+      { showBanner ? <LotBanner /> : null }
```

Comment:
I believe that you are getting mixed up about the LotBanner, etc.
The LotBanner is a display that shows a single Lot from firestore. So it is meant to format and present the screenshot, the dollar amount bid, the pickup, etc.
When the user clicks the "Where to?" button, it should present them with the LotSubmissionForm. This allows them to submit the screenshot, the amount, etc. These go to Firestore, and then they can be accessed from the DriverHome, which displays the LotBanner.
So the PassengerHome is doing two things:
1, It can take you to the LotSubmissionForm, where you write a lot and send it to firestore
2, Edison, you don't need to worry about this right now, but when a passenger is matched with a driver, it should show this, and what the time left is, what the price is, etc.